packages/grafana-data/src/types/featureToggles.gen.ts

```diff
@@ -47,4 +47,5 @@ export interface FeatureToggles {
   annotationComments?: boolean;
   migrationLocking?: boolean;
   fileStoreApi?: boolean;
+  resourcePickerForMetrics?: boolean;
```

Comment:
lol everybody is using a different case here (camelCase, snake_case, kebab-case...). Anyway, I would put "azure" somewhere in the name so it's clear its scope.